multicluster/controllers/multicluster/leader_clusterset_controller.go

```diff
@@ -83,7 +83,7 @@ func (r *LeaderClusterSetReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	klog.InfoS("Received ClusterSet add/update", "clusterset", klog.KObj(clusterSet))
 
 	// Handle create or update
-	// If create, make sure the local ClusterClaim is part of the leader config
+	// If create, make sure the local ClusterClaim is part of the leader ClusterSet.
```

Comment:
What does "ClusterClaim is part of the leader ClusterSet" mean?

Reply:
The following codes are validating local ClusterClaims are defined and the cluster/ClusterSet IDs are also defined in the leader ClusterSet CR. I have refined this comment.